tests/conftest.py

```diff
@@ -163,17 +162,18 @@ def entities(client):
 
             db.session.add(r)
 
-        # Cavatica task genomic files
+        # Cavatica task genomic files and read group
         for i, (ct, gf) in enumerate(zip(
                 _entities[CavaticaTask], _entities[GenomicFile])):
             is_input = True
             if i % 2 == 0:
                 is_input = False
+                rg = ReadGroup(genomic_file_id=gf.kf_id)
```

Comment:
Remove these 3 lines here. Read groups will get created above in the main entity creation loop (unless you were to put it in the list of skipped entities which currently has FamilyRelationship and CavaticaTaskGenomicFile). Fam rels and ctgfs are special entities (association tables) that have to be created after all the main entities are created since they require two entities before they can be created. Fam rel needs two participants, Ctgf needs cav task and genomic file.